lib/fpm/package/python.rb

```diff
@@ -228,6 +231,10 @@ def load_package_info(setup_py)
         end
         name, cmp, version = match.captures
 
+        if attributes[:python_disable_dependencies]
+          next if attributes[:python_disable_dependencies].include?(name)
```

Comment:
the outer `if` here can be removed, I think, if you set the `:default => []` in the option